demo/api/src/importer/data-streams/data-stream.ts

```diff
@@ -0,0 +1,11 @@
+export interface DataStreamAndMetadata {
+    dataStream: NodeJS.ReadableStream;
+    params?: Record<string, unknown>;
+    additionalData?: Record<string, unknown>;
+}
+
+export type DataStreamClass = { new (): DataStream };
+
+export abstract class DataStream {
+    abstract getDataStreamsAndMetadata(): Promise<DataStreamAndMetadata | null>;
```

Comment:
Is it data stream**s** or data stream? The type is singular, but the method is plural

Reply:
It should be singular